docs/api/page/README.md

```diff
@@ -19,3 +19,9 @@
 > Function used by [`fieldResolver`](resolver/fieldResolver.md) to generate `input` fields.
 
 [`View source`](../../../src/page/resolver/field/inputGenerator.js)
+
+## [`page/formHandler/formHandler`](formHandler/formHandler.md)
+
+> Class which is creating function used to handle form submission.
+
+[`View source`](../../../src/page/formHandler/formHandler.js)
```

Comment:
The function is called on form submission, it sends a request with form data and returns success or failure action, right? First of all why do we need it?

Reply:
because we have to raise actions so we will be able to handle this events in app's `redux-saga`